tests/test_repository.py

```diff
@@ -0,0 +1,49 @@
+import os
+import pytest
+
+from typer.testing import CliRunner
+from typer import get_app_dir
+from tests.conftest import get_config_file
+import thipstercli.constants as constants
+from thipstercli.repository import repository_app
+
+runner = CliRunner(mix_stderr=False)
+
+
+@pytest.fixture
+def create_models_directory():
+    models_path = os.path.join(get_app_dir(constants.APP_NAME), 'models')
+    os.mkdir(models_path)
+
+    yield models_path
+
+    if os.path.exists(models_path):
+        os.rmdir(models_path)
```

Comment:
https://beta.ruff.rs/docs/rules/#flake8-use-pathlib-pth
Replace:  
- os.path.join with Path(get_app_dir(constants.APP_NAME)) / 'models'
- os.mkdir(models_path) with Path(models_path).mkdir()
- os.rmdir(models_path) with Path(models_path).rmdir()

Reply:
Rule wasn't in ruff config btw, just added it